tools/travis-scripts/run-script.sh

```diff
@@ -41,24 +41,21 @@ function build_android()
 {
     # Build all samples
     echo "Building Android samples ..."
-    export COCOS_CONSOLE_ROOT=$COCOS2DX_ROOT/tools/cocos2d-console/bin
-    export ANDROID_SDK_ROOT=/usr/local/android-sdk
-    export COCOS_X_ROOT=$COCOS2DX_ROOT
-    export PATH=$ANDROID_SDK_ROOT:$COCOS_CONSOLE_ROOT:$PATH
+    source ../environment.sh
 
     # build cpp-empty-test
     # pushd $COCOS2DX_ROOT/tests/cpp-empty-test
     # cocos compile -p android --android-studio
     # popd
 
     # build cpp-tests
-    pushd $COCOS2DX_ROOT/tests/cpp-tests
-    cocos compile -p android
+    pushd $COCOS2DX_ROOT/tests/cpp-tests/proj.android
+    ./gradlew build
     popd
 
     # build lua-tests
-    pushd $COCOS2DX_ROOT/tests/lua-tests
-    cocos compile -p android
+    pushd $COCOS2DX_ROOT/tests/lua-tests/project/proj.android
+    ./gradlew build
```

Comment:
I found the android build always TimeOut, this task build the engine for every test project build. it build `cpp-tests` and `lua-tests`, spending 2 times the build engine time.

Can we divide this task into 2 travis task? I think TimeOut problem will be solved after divide.

Reply:
I change it to "./gradlew assembleDebug". "cocos compile -p android" built debug only.
"./gradlew build" builds Debug and Release